aea/package_manager/base.py

```diff
@@ -277,8 +372,15 @@ def json(
     ) -> OrderedDictType:
         """Json representation"""
         data = OrderedDict()
-        for package_id, package_hash in self._packages.items():
-            data[package_id.to_uri_path] = package_hash
+        data["dev"] = OrderedDict()
+        data["third_party"] = OrderedDict()
```

Comment:
not needed anymore since python 3.7, but I like this because it's explicit (second line of the Zen of Python)